src/components/action-bar/action-bar-item/action-bar-item.tsx

```diff
@@ -105,11 +110,14 @@ export class ActionBarButton {
         }
 
         if ('icon' in this.item) {
+            const name = getIconName(this.item.icon);
+            const color = getIconColor(this.item.icon, this.item.iconColor);
+
             return (
                 <limel-icon
-                    name={this.item.icon}
+                    name={name}
                     style={{
-                        '--action-bar-item-icon-color': `${this.item.iconColor}`,
+                        '--action-bar-item-icon-color': `${color}`,
```

Comment:
🤔 now if the `iconColor` is defined (the old way of doing it), the `--action-bar-item-icon-color` becomes `undefined`. 

<img width="950" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/c595c777-5de7-4209-ae64-a172ead897ee">